Sources/Amplitude/Amplitude.m

```diff
@@ -1250,7 +1250,7 @@ - (void)enterBackground {
 
     // Stop uploading
     [self endBackgroundTaskIfNeeded];
-    _uploadTaskID = [app beginBackgroundTaskWithExpirationHandler:^{
+    _uploadTaskID = [app _uploadTaskID = [app beginBackgroundTaskWithName:@"Amplitude: enterBackground" expirationHandler::^{
```

Comment:
@antohisorin Looks like this does not compile? Shoudl this be

```
    _uploadTaskID = [app beginBackgroundTaskWithName:@"Amplitude: enterBackground" expirationHandler:^{
```

Reply:
Updated.